miso-web/src/main/webapp/scripts/list_samples.js

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+ListTarget.sample = {
+  name : "Sample",
+  createUrl : function(config, projectId) {
+    return "/miso/rest/tree/samples/dt" + (projectId ? '/project/' + projectId
+        : '');
+  },
+  createBulkActions : function(config, projectId) {
+    var actions = HotTarget.sample.bulkActions;
+    
+    if (projectId) {
+      actions = actions.concat([
+          {
+            name : "Get Form",
+            action : function(items) {
+              Utils.showDialog("Get Information Form", "Next", [ {
+                property : "format",
+                type : "select",
+                label : "Type",
+                values : [ {
+                  name : "Tubes",
+                  value : false
+                }, {
+                  name : "Plate",
+                  value : true
+                } ],
+                getLabel : Utils.array.getName
+              } ], function(result) {
+                Project.ui.processSampleDeliveryForm(projectId,
+                    result.format.value, items.map(Utils.array.getId));
+              });
+            }
+          }, {
+            name : "Receive",
+            action : function(items) {
+              Project.ui.receiveSelectedSamples(items.map(Utils.array.getId));
+            }
+          }, ]);
+    }
+    return actions;
+    
+  },
+  createStaticActions : function(config, projectId) {
+    return [
+        {
+          name : "Create",
```

Comment:
The "Lookup Identities" button is indeed gone. I'll make the correction to make the external name look required; the Sex column indicates it's required in #964 but I can change that to defaulting to Unknown. (It gets cranky that "" aka empty string isn't in the Sex dropdown list, since it's represented by Unknown.)